extensions/reactive-routes/runtime/src/main/java/io/quarkus/vertx/web/ReactiveRoutes.java

```diff
@@ -12,6 +12,74 @@
  */
 public class ReactiveRoutes {
 
+    /**
+     * The content-type to use to indicate you want to produce an JSON Array response, such as in:
+     * 
+     * <pre>
+     * {@code
+     * &#64;Route(path = "/heroes", produces = ReactiveRoutes.APPLICATION_JSON_CONTENT_TYPE)
+     * Multi<Person> heroes() {
+     *     return Multi.createFrom().items(
+     *             new Person("superman", 1),
+     *             new Person("batman", 2),
+     *             new Person("spiderman", 3));
+     * }
+     * }
+     * </pre>
+     *
+     * Note that the array is streamed object per object.
+     * Each object is written individually in the response, until the last one.
+     */
+    public static final String APPLICATION_JSON_CONTENT_TYPE = "application/json";
```

Comment:
Doesn't vert.x web define some ContentType constants? Anyway, I think that we could use a shorter name such as `APPLICATION_JSON`.

Reply:
No there are no constants we can reuse, but I've reduced the name length.